src/mca/pdl/pdlopen/pdl_pdlopen_module.c

```diff
@@ -175,7 +175,26 @@ static int pdlopen_foreachfile(const char *search_path,
 
         dp = opendir(dirs[i]);
         if (NULL == dp) {
-            ret = PMIX_ERR_IN_ERRNO;
+            if (0 == i) {
```

Comment:
If my understanding is correct we are ok if some of the directories are missing:
```shell
PMIXP_INSTALL_PREFIX="x:y:z"
```
As it is implemented now - you are failing if `x` can't be opened. But what if `x` is missing, but `y` is what we need?